lib/grammers-session/src/message_box/adaptor.rs

```diff
@@ -456,6 +458,14 @@ impl PtsInfo {
                 pts_count: 0,
                 entry: Entry::SecretChats,
             }),
+            MessageReactions(_) => None,
+
+            // LAYER 143
```

Comment:
I don't think separating these with a comment is necessary. We can remove the comment and leave the new arms together with the rest.

Reply:
Removed that